src/main/webapp/web.xml

```diff
@@ -3,106 +3,12 @@
          xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee
                       http://xmlns.jcp.org/xml/ns/javaee/web-app_4_0.xsd"
          version="4.0">
-    <servlet>
-        <servlet-name>index</servlet-name>
-        <servlet-class>taxi.controller.IndexController</servlet-class>
-    </servlet>
-    <servlet-mapping>
-        <servlet-name>index</servlet-name>
```

Comment:
![image](https://github.com/mate-academy/jv-web-security/assets/126912806/bc84331f-04c7-4e4e-a798-a1551505881a)
Because I get a "page not found" error when I use mapping, . When I use annotation everything works fine for me. I don't know why this happens